pkg/arvo/sur/rw-security.hoon

```diff
@@ -0,0 +1,8 @@
+|%
++$  rw-security
```

Comment:
Can we please not do unrelated renaming/restructuring in the same commits as feature changes?

Reply:
I think this is a small, positive change...

Reply:
And I agree! Just saying it's bad practice to include it in a commit that does something completely unrelated. The change isn't at all detectable from the commit message, and it pollutes the diff of the change the commit is claiming to make, which is messier for review.

Not a big deal in this particular case, but gotta stay vigilant. (:

Reply:
Separate commits endorsed.

Reply:
I have now split this up into different commits. Laborious, but worth it!

Reply:
Positively virtuous!